src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -1,40 +1,51 @@
 import {
-  AccountInformationIF, AlterationFilingIF, CertifyIF, DefineCompanyIF, NameRequestIF, TombStoneIF, PeopleAndRolesIF,
-  ShareStructureIF, EffectiveDateTimeIF, IncorporationAgreementIF, BusinessInformationIF, IncorporationFilingIF,
+  AccountInformationIF, AlterationFilingIF, CertifyIF, NameRequestIF, TombStoneIF, PeopleAndRolesIF, ShareStructureIF,
+  EffectiveDateTimeIF, IncorporationAddressIf, IncorporationAgreementIF, BusinessInformationIF, IncorporationFilingIF,
   FilingDataIF, StaffPaymentStepIF, DetailIF, NameTranslationIF, BusinessSnapshotIF, DocumentDeliveryIF, FeesIF,
   NewAlterationIF
 } from '@/interfaces'
+import { ContactPointIF } from '@bcrs-shared-components/interfaces'
 
 /** Data object used internally only (not to/from API). */
 export interface StateModelIF {
   currentJsDate: Date
   tombstone: TombStoneIF
+  newAlteration: NewAlterationIF
   accountInformation: AccountInformationIF
+  businessContact: ContactPointIF
   businessInformation: BusinessInformationIF
   nameRequest: NameRequestIF
   nameTranslations: NameTranslationIF[]
   certifyState: CertifyIF
   documentDelivery: DocumentDeliveryIF
-  defineCompanyStep: DefineCompanyIF
+  officeAddresses: IncorporationAddressIf | {}
```

Comment:
I don't think so, because this is a declaration not an assignment. This means that officeAddresses can be of either type IncorporationAddressIf or an empty object.

My preference would be to remove the empty object spec and use "null" when it's not set, but misc code would have to be updated to handle null dereferences (eg, `officeAddresses.foo` will fail if it's null, but I think will be undefined if it's {}).

Anyway, not my code... I just moved officeAddresses out of defineCompanyStep and directly here (so that it is filing-agnostic).

Reply:
This is a "union type". See https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#working-with-union-types

Reply:
oh, yes... my misunderstanding. Thanks!